spec/controllers/catalog_controller_spec.rb

```diff
@@ -535,6 +535,7 @@ def export_as_mock
         post :email, xhr: true, params: { id: doc_id, to: 'test_email@projectblacklight.org' }
         expect(request).to render_template 'email_success'
         expect(request.flash[:success]).to eq "Email Sent"
+        expect(flash.instance_variable_get("@discard").first).to eq("success")
```

Comment:
This seems like a fragile test because it is tightly coupled to the implementation.  A better way might do another `get` request and show that the `request.flash[:success]` is empty.

Reply:
Thanks @jcoyne, good point, I've changed the tests.